.github/workflows/e2e-advanced-tests.yml

```
@@ -0,0 +1,296 @@
+          BREVO_API_KEY: ${{ secrets.BREVO_TEST_API_KEY }}
+        run: |
+          # Create .env.local for E2E tests
+          cat > .env.local << EOF
+          TURSO_DATABASE_URL=$TURSO_DATABASE_URL
+          TURSO_AUTH_TOKEN=$TURSO_AUTH_TOKEN
+          STRIPE_PUBLISHABLE_KEY=$STRIPE_PUBLISHABLE_KEY
+          STRIPE_SECRET_KEY=$STRIPE_SECRET_KEY
+          BREVO_API_KEY=$BREVO_API_KEY
+          E2E_TEST_MODE=true
+          EOF
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)